specification/compute/resource-manager/readme.md

```diff
@@ -235,6 +235,49 @@ directive:
       - RequiredPropertiesMissingInResourceModel
 ```
 
+### Tag: package-2022-01-03
+
+These settings apply only when `--tag=package-2022-01-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2022-01-03'
+input-file:
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineScaleSet.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachine.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineImage.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/virtualMachineExtensionImage.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/availabilitySet.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/proximityPlacementGroup.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/sshPublicKey.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/image.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/restorePoint.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/capacityReservation.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/logAnalytic.json
+- Microsoft.Compute/stable/2022-03-01/ComputeRP/runCommand.json 
+- Microsoft.Compute/stable/2021-07-01/skus.json
+- Microsoft.Compute/stable/2021-12-01/disk.json
+- Microsoft.Compute/stable/2022-01-03/common.json
+- Microsoft.Compute/stable/2022-01-03/GalleryRP/galleryRPCommon.json
+- Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json
+- Microsoft.Compute/stable/2022-01-03/GalleryRP/sharedGallery.json
+- Microsoft.Compute/stable/2022-01-03/GalleryRP/communityGallery.json
+- Microsoft.Compute/stable/2021-03-01/cloudService.json
+```
```

Comment:
Adding this tag which is definitely newer than `package-2022-03-01`, but considering its name, it gives us an illusion that this is older than `package-2022-03-01`.
Could we just replace the old gallery with the new ones introduced by this PR in `package-2022-03-01` and remove this tag to avoid this?

Reply:
If no SDKs have been released with 2022-03-01, then yes we can do that. I know we haven't requested any SDK releases.

Reply:
So I am reticent in updating 2022-03-01.

Reply:
@ArcturusZhang , and we have now already generated SDKs on the 2022-03-01 version, so we now cannot edit 2022-03-01 tags.